src/core/dpd.hpp

```diff
@@ -54,5 +60,49 @@ Utils::Vector3d dpd_pair_force(Particle const *p1, Particle const *p2,
                                const IA_parameters *ia_params, double const *d,
                                double dist, double dist2);
 Utils::Vector9d dpd_stress();
+
+/** philox functiontality: increment, get/set */
+extern std::unique_ptr<Utils::Counter<uint64_t>> dpd_rng_counter;
```

Comment:
I think all of the philox and counter stuff can be moved to `dpd.cpp`, as far as I can tell it's not used elsewhere.